concepts/modularization/README.md

```diff
@@ -0,0 +1,83 @@
+# Modularization
+
+Kyma provides Kubernetes building blocks. It should be easy to pick only those that are needed for the job and it should be easy to add new blocks to extend Kyma features. With the growing number of components, it is not possible to always install them all anymore. 
+
+- [Independent releases of modules](#independent-releases-of-modules)
+- [Dependencies between components](#dependencies-between-components)
+- [Release channels](#release-channels)
+- [Versioning](#versioning)
+- [Manifest operator](#manifest-operator)
+
+# Independent releases of modules
+With the growing number of components, it is hard to deliver features and fixes quickly and efficiently. Changes in manifests require a new release of Kyma. Operators (reconcilers) are tightly coupled and must be released together. In most cases, new component releases don't involve any API changes and could be delivered in a few minutes. 
+
+# Dependencies between components
+Components can depend only on core Kubernetes API or API extensions introduced by other components. Component operators are responsible for checking if the required APIs are available and react properly to the missing dependencies by reducing functionality or even reporting errors. Component owners are responsible for integration tests with their dependencies (with all versions supported in official release channels).
+
+**Example:**
+
+If the API you require is not available you should fail (e.g. core kubernetes API or istio virtual service). If your component can work without the API, but some features are not available (e.g. service monitor from monitoring) you should just skip it and continue to deploy other component resources. 
+
+# Release channels
+Release channels let customers balance between available features and stability. The number of channels and their characteristics can be established later. Usually, projects introduce between 2 and 4 channels. Some examples:
```

Comment:
How release channels will be tied What's new articles published on SAP Help. Currently we have really big issue with communication about planned changes to our customers. With Modules it will be even greater.  

As a customer I need to get a heads up about planned changes a few weeks before its released so that  I can prepare for the change. Currently we barely can provide such info (indeed we give a heads up but without concrete date it will be available).

Reply:
With release channels, you can see what component versions are in the rapid channel and you can even use test your application with the component version coming from other channel. And the idea behind independent releases of modules is that it is no more big bang releases with multiple components and significant changes.